src/posthog-core.ts

```diff
@@ -272,6 +270,12 @@ export class PostHog {
     SentryIntegration: typeof SentryIntegration
     segmentIntegration: () => any
 
+    /** DEPRECATED: We keep this to support existing usage but now one should just call .identify */
+    people: {
+        set: (prop: string | Properties, to?: string, callback?: RequestCallback) => void
+        set_once: (prop: string | Properties, to?: string, callback?: RequestCallback) => void
+    }
+
```

Comment:
We might want to keep these as is because `$identify` calls trigger more complex code paths (as in not deprecate them and not suggest calling .identify see other comments)

The current code doesn't set `is_identified` to be `true` when `$identify` is used (isn't that confusing), because we don't reach [here](https://github.com/PostHog/posthog/blob/016c49cb41d9dd708b55de287bb466a4996303c2/plugin-server/src/worker/ingestion/person-state.ts#L357) if the [ids match](https://github.com/PostHog/posthog/blob/016c49cb41d9dd708b55de287bb466a4996303c2/plugin-server/src/worker/ingestion/person-state.ts#L321), but I don't want to assume that it doesn't as that's very confusing and we might change that in the future.